### DISCUSSION TITLE
Passkey window very slow to show up

### DISCUSSION THREAD

**Discussion by laurentftech** [dani-garcia/vaultwarden#4068] [General]
Hi all,
I've got Vaultwarden (v 2023.10.0) running on docker on my Synology drive (DS420+) behind caddy at home. 
I noticed that the passkey window is very slow to show up (around 15s and sometimes more).
I just tested it with a Bitwarden.com test account (with few passwords though) and the passkey window show up is <2s.
Do you see that too ?
Thanks.

**Comment by Node815** [comment]
Mine is anywhere from 2s to ~5s  .  I do have quite a few containers on my server, so I don't know if it's based on the server load or my computer's resource usage (Arch Linux), so it may be that too.   I also have VW behind Cloudflare tunnels so I am technically hopping from my local computer through the web traversing back through their tunnel to my server at my feet under my desk. :)  So I have to figure in a little bit of latency.

**Reply by laurentftech** [reply]
According to https://github.com/dani-garcia/vaultwarden/discussions/3355#discussioncomment-7610313 it is a client performance issue. 
I have a lot of passwords (>1000) so it may be related. Waiting for a MacOS client update.
